packages/dialtone-tokens/build-sd-transforms.js

```diff
@@ -160,7 +160,6 @@ export async function run () {
         },
         android_compose: {
           transforms: [
-            'ts/resolveMath',
```

Comment:
Why was this removed?

Reply:
It's only supported in javascript. I think I put it in there previously to see if it would work on android and then forgot to remove it.